### ISSUE TITLE
Cstick Down and tilt stick up don't do dash attack during initial dash, and don't register until late into initial dash

### ISSUE THREAD

**Issue by sparkswordkirby** [HDR-Development/HewDraw-Remix#1081]
When inputting cstick down with tilt/smash stick or cstick up with tilt stick during initial dash, you will do those respective moves instead of dash attack even when holding forward the entire time. The input is also not registered until doing these inputs near the end of initial dash. This is not the case for c-stick left or right, which can do dash attack from the very first frame of initial dash.

As far as I'm aware, no other smash game has this issue and it makes doing moves like DACUS and DACDS a little awkward for people used to that input method from the older brawl days (a well known input for DACUS was c-stick down + up Z)

TL;DR:
- down tilt, down smash, and up tilt when done with c-stick do not do dash attack when holding forward during initial dash
- these moves' inputs also do not get accepted at beginning of initial dash

Replication: Attempt to do down tilt, down smash, or up tilt with the c stick right at the beginning of initial dash(to see the input not get read), and then near the end of initial dash(to see it do the move instead of dash attack)
